runtime/compute/cker/include/cker/Shape.h

```diff
@@ -47,49 +47,22 @@ class Shape
   // Constructor that takes a dimension count.
   // If dimensions_count <= kMaxSmallSize, it uses a fixed-size array.
   // Otherwise, it uses a dynamic vector.
-  explicit Shape(int dimensions_count) : _size(dimensions_count)
-  {
-    if (dimensions_count <= kMaxSmallSize)
-    {
-      dims_ = std::array<int32_t, kMaxSmallSize>{};
-    }
-    else
-    {
-      dims_ = std::vector<int32_t>(dimensions_count);
-    }
-  }
+  explicit Shape(int dimensions_count) : _size(dimensions_count) { initStorage(dimensions_count); }
 
   // Constructor that creates a shape of given size and fills all dimensions with "value".
-  Shape(int shape_size, int32_t value) : _size(0)
+  Shape(int shape_size, int32_t value) : _size(shape_size)
```

Comment:
This generated test errors because this constructor did not apply `shape_size` to `_size` anywhere.